Default.sublime-keymap

```diff
@@ -1,4 +1,6 @@
 [
+    // To change the keybind of a command, copy its section, and edit the 'keys' part
```

Comment:
If we follow the ST nomenclature then each object is called a "key binding" and we are calling it "command" here which is rather just a part of a key binding. So this sentence can be confusing.

Maybe:

```suggestion
    // To enable or change an existing key binding, copy it to the right view.
    // To change the associated key press, edit the "keys" value.
```